win32ss/user/user32/misc/exticon.c

```diff
@@ -814,9 +848,16 @@ UINT WINAPI PrivateExtractIconExW (
 	TRACE("%s %d %p %p %d\n",
 	debugstr_w(lpwstrFile),nIndex,phIconLarge, phIconSmall, nIcons);
 
-	if (nIndex == -1)
-	  /* get the number of icons */
-	  return ICO_ExtractIconExW(lpwstrFile, NULL, 0, 0, 0, 0, NULL, LR_DEFAULTCOLOR);
+#ifdef __REACTOS__
+    if (nIndex == -1 || (!phIconSmall && !phIconLarge))
+      /* get the number of icons */
+      return ICO_ExtractIconExW(lpwstrFile, NULL, 0, 0, 0, 0, NULL,
+                                LR_DEFAULTCOLOR, FALSE);
+#else
+    if (nIndex == -1)
```

Comment:
Lines in the `#else` should not show up as changed in the diff. I think you changed some spacing here (e.g. tabs to spaces)?

Reply:
Yes, I had converted most tabs to spaces. These changes should not be reverted. Thanks.